content/content-strategy/audit-content/plan-your-audit/index.yml

```diff
@@ -0,0 +1,24 @@
+pagetitle: Plan your audit
+description: Abstract
```

Comment:
`description` needs to be revised soon. 'Abstract' does not meet metadata requirements.

Reply:
Trisha fixed